openvino_xai/explainer/explainer.py

```diff
@@ -108,18 +108,25 @@ def create_method(self, explain_mode: ExplainMode, task: Task) -> MethodBase:
     def __call__(
         self,
         data: np.ndarray,
-        explanation_parameters: ExplanationParameters,
+        target_explain_group: TargetExplainGroup = TargetExplainGroup.CUSTOM,
+        target_explain_labels: List[int | str] | None = None,
+        label_names: List[str] | None = None,
+        scaling: bool = False,
+        resize: bool = True,
+        colormap: bool = True,
+        overlay: bool = False,
+        overlay_weight: float = 0.5,
```

Comment:
This make sense, but it incorporates some logic inside, this might be a bit confusing to grasp right away by the user. I would propose to keep it as is.